specutils/spectra/spectrum1d.py

```diff
@@ -91,17 +92,54 @@ def to_flux(self, unit):
             The converted flux array.
         """
         new_data = self.flux.to(
-            unit, equivalencies=eq.spectral_density(self.dispersion))
+            unit, equivalencies=eq.spectral_density(self.spectral_axis))
 
         self._data = new_data.value
         self._unit = new_data.unit
 
         return self.flux
 
-    def to_dispersion(self, unit, rest, type="doppler_relativistic"):
+    @property
+    def frequency(self):
+        return self._spectral_axis.to(u.GHz, u.spectral())
+
+    @property
+    def wavelength(self):
+        return self._spectral_axis.to(u.AA, u.spectral())
```

Comment:
While I prefer AA and would use it most of all, I propose this should
be set above,  eg.  self.wavelength_unit  (or self.wv_unit) and defaulted
there as u.AA.

Same for other units, I suppose